server/plugin/virtual_agent_test.go

```diff
@@ -344,6 +351,13 @@ func Test_CreateDefaultDateAttachment(t *testing.T) {
 						},
 						Type: "button",
 					},
+					{
+						Name: Skip,
+						Type: "button",
```

Comment:
Check this everywhere
```suggestion
						Type: model.POST_ACTION_TYPE_BUTTON,
```